packages/genui/openui/src/core/renderer.css

```
@@ -5,6 +5,42 @@
+.OpenUIRendererContent {
+  width: 100%;
+  opacity: 1;
+}
+
+.OpenUIRendererContentLoading {
+  opacity: 0.72;
+}
+
+.OpenUIQueryLoader {
+  position: absolute;
+  top: 8rpx;
```

Stylelint (17.12.0)

[error] Unknown unit "rpx" (unit-no-unknown)

(unit-no-unknown)